services/dis-vault-operator/.devcontainer/post-install.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -x
+
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-linux-$(go env GOARCH)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)